<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Editor</class>
 <widget class="QMainWindow" name="Editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cotorro Editor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="app-resources.qrc">
    <normaloff>:/icons/assets/icons/cotorro.ico</normaloff>:/icons/assets/icons/cotorro.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QSplitter" name="mainSplitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="opaqueResize">
       <bool>true</bool>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="layout_storytellingSection" stretch="0,0">
        <property name="spacing">
         <number>3</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetMaximumSize</enum>
        </property>
        <property name="leftMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QGroupBox" name="group_storySection">
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>Story Sections</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="btn_addSection">
                 <property name="text">
                  <string>Add</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btn_removeSection">
                 <property name="text">
                  <string>Remove</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QListWidget" name="list_storySections">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>1600000</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="wordBox">
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>Story Section Words</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QListWidget" name="list_storySectionWords"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="childrenCollapsible">
           <bool>false</bool>
          </property>
          <widget class="QGroupBox" name="panelStorySectionEditor">
           <property name="title">
            <string>Story Section Editor</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Name:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineSectionName"/>
              </item>
              <item>
               <widget class="QPushButton" name="btnRename">
                <property name="text">
                 <string>Rename</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Audio:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lblAudioKey">
                <property name="text">
                 <string>Audio Name Here</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QPushButton" name="btnPlaySimulation">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="app-resources.qrc">
                  <normaloff>:/icons/assets/icons/player_play.ico</normaloff>:/icons/assets/icons/player_play.ico</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnPauseSimulation">
                <property name="icon">
                 <iconset resource="app-resources.qrc">
                  <normaloff>:/icons/assets/icons/player_pause.ico</normaloff>:/icons/assets/icons/player_pause.ico</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnStopSimulation">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="app-resources.qrc">
                  <normaloff>:/icons/assets/icons/player_stop.ico</normaloff>:/icons/assets/icons/player_stop.ico</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lblVolumen">
                <property name="text">
                 <string>volume:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="sliderVolumen">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximum">
                 <number>100</number>
                </property>
                <property name="singleStep">
                 <number>5</number>
                </property>
                <property name="value">
                 <number>100</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="tickInterval">
                 <number>1</number>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>zoom:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="sliderZoom">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Active Word:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lblWordKey">
                <property name="text">
                 <string>Active Word Here</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Start:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_wordStart"/>
              </item>
              <item>
               <widget class="QPushButton" name="btnStartKey">
                <property name="text">
                 <string>Start</string>
                </property>
                <property name="icon">
                 <iconset resource="app-resources.qrc">
                  <normaloff>:/icons/assets/icons/security_key.ico</normaloff>:/icons/assets/icons/security_key.ico</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>End</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_wordEnd"/>
              </item>
              <item>
               <widget class="QPushButton" name="btnEndKey">
                <property name="text">
                 <string>End</string>
                </property>
                <property name="icon">
                 <iconset resource="app-resources.qrc">
                  <normaloff>:/icons/assets/icons/security_key.ico</normaloff>:/icons/assets/icons/security_key.ico</iconset>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_5">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QGroupBox" name="groupSFMLCanvas">
              <property name="title">
               <string>Editor</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="group_logger">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>120</height>
           </size>
          </property>
          <property name="title">
           <string>Logger</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QPlainTextEdit" name="pText_logger">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuSections">
    <property name="title">
     <string>Sections</string>
    </property>
    <addaction name="actionAddSection"/>
    <addaction name="actionRemoveSection"/>
   </widget>
   <widget class="QMenu" name="menuSimulation">
    <property name="title">
     <string>Simulation</string>
    </property>
    <addaction name="actionPlaySimulation"/>
    <addaction name="actionPauseSimulation"/>
    <addaction name="actionStopSimulation"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSections"/>
   <addaction name="menuSimulation"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAddSection"/>
   <addaction name="actionRemoveSection"/>
   <addaction name="separator"/>
   <addaction name="actionPlaySimulation"/>
   <addaction name="actionPauseSimulation"/>
   <addaction name="actionStopSimulation"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset>
     <normalon>:/icons/assets/icons/file.ico</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>New project.</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="app-resources.qrc">
     <normaloff>:/icons/assets/icons/folder.ico</normaloff>:/icons/assets/icons/folder.ico</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open project.</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="app-resources.qrc">
     <normaloff>:/icons/assets/icons/diskette.ico</normaloff>:/icons/assets/icons/diskette.ico</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save project.</string>
   </property>
  </action>
  <action name="actionAddSection">
   <property name="icon">
    <iconset resource="app-resources.qrc">
     <normaloff>:/icons/assets/icons/document_add.ico</normaloff>:/icons/assets/icons/document_add.ico</iconset>
   </property>
   <property name="text">
    <string>Add Section</string>
   </property>
   <property name="toolTip">
    <string>Add a story section.</string>
   </property>
  </action>
  <action name="actionRemoveSection">
   <property name="icon">
    <iconset resource="app-resources.qrc">
     <normaloff>:/icons/assets/icons/document_delete.ico</normaloff>:/icons/assets/icons/document_delete.ico</iconset>
   </property>
   <property name="text">
    <string>Remove Section</string>
   </property>
   <property name="toolTip">
    <string>Remove a story section.</string>
   </property>
  </action>
  <action name="actionPlaySimulation">
   <property name="icon">
    <iconset resource="app-resources.qrc">
     <normaloff>:/icons/assets/icons/player_play.ico</normaloff>:/icons/assets/icons/player_play.ico</iconset>
   </property>
   <property name="text">
    <string>Play Simulation</string>
   </property>
   <property name="toolTip">
    <string>Play simulation.</string>
   </property>
  </action>
  <action name="actionPauseSimulation">
   <property name="icon">
    <iconset resource="app-resources.qrc">
     <normaloff>:/icons/assets/icons/player_pause.ico</normaloff>:/icons/assets/icons/player_pause.ico</iconset>
   </property>
   <property name="text">
    <string>Pause Simulation</string>
   </property>
   <property name="toolTip">
    <string>Pause Simulation.</string>
   </property>
  </action>
  <action name="actionStopSimulation">
   <property name="icon">
    <iconset resource="app-resources.qrc">
     <normaloff>:/icons/assets/icons/player_stop.ico</normaloff>:/icons/assets/icons/player_stop.ico</iconset>
   </property>
   <property name="text">
    <string>Stop Simulation</string>
   </property>
   <property name="toolTip">
    <string>Stop simulation.</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="app-resources.qrc"/>
 </resources>
 <connections/>
</ui>
